angular.json

```diff
@@ -33,7 +33,12 @@
             "styles": [
               "src/styles.scss"
             ],
-            "scripts": []
+            "scripts": [],
+            "stylePreprocessorOptions": {
```

Comment:
adding this enables the use of the relative path on component styles `*.component.scss`
so we can import the variable and functions like this without worry about the path
```scss
@import 'functions/sizing';
@import 'variables';
```
for more info about why we cannot use global variables on component styles please refer to [this issue](https://github.com/angular/angular-cli/issues/1253)